web_programming/instagram_crawler.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+import requests
+from bs4 import BeautifulSoup
+import json
+
+headers = \
+    {
+        'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36'}
+
+# Usage
+"""
+>>> user = Instagram("github")
+>>> user.is_verified()
+True
+>>> user.get_biography()
+Built for developers.
+
+"""
+
+
+class Instagram(object):
+    """
+    Class Instagram crawl instagram user information
+    """
+
+    def __init__(self, username):
+        self.username = username
+        self.url = 'https://www.instagram.com/{}/'.format(username)
```

Comment:
```suggestion
        self.url = f'https://www.instagram.com/{username}/'
```
As discussed in CONTRIBUTING.md, please use f-strings where they make sense.

Reply:
Do we really want to query the back end (instagram) for every one of the fields below?  Most of these values are very static and are unlikely to change minute-to-minute.  Perhaps it would be better to have a `self.user_data` dict that contained the results of self.get_json() and the other methods could use that data.